components/PlayerButtons/PlayerButtons.tsx

```diff
@@ -11,33 +12,28 @@ type PlayerButtonProps = {
 };
 
 const PlayerButtons = ({ metadata }: PlayerButtonProps) => {
+  const { hash, songLink } = useParams();
   const [player, dispatch] = usePlayer();
   const { handleNext, handlePrev } = useFeedProvider();
+  const id = metadata?.id;
+  const showActions = !(hash || songLink);
 
   useEffect(() => {
     if (player.position !== 0 && player.position >= player.duration) {
```

Comment:
optional: would be helpful to reassign this to a variable `songHasEnded` to be easier to maintain.